modules/rosa-architecture.adoc

```diff
@@ -0,0 +1,13 @@
+[id="rosa-architecture_{context}"]
+= ROSA architecture on public and private networks
+
+You can install ROSA using either a public or private network.
+
+A private cluster and private network connection can be configured for ROSA during the cluster creation process.
+
+In either the public network or private network scenario, Red Hat SREs manage the control plane and infrastructure nodes with limited access through a public network. Red Hat SREs do not access or manage the customer’s worker nodes. For more information, see the Service Definition.
```

Comment:
Do you think we need to mention our roles like SREs? I think Red Hat is sufficient. Also, please link to Service Definition.

Reply:
I think in line 8 we should reconsider the phrasing that says that we do not manage the customers worker nodes.  The messaging we share with customers is that we do manage them, we support them and proactively monitor them to prevent issues. This removes some of the differentiation between xKS and us.  Maybe we should change to say "Red Hat SREs manage the cluster with limited access through a public network." And remove the do not manage worker nodes line.

Reply:
Done. Although we cannot add cross-reference links in modules at this time, it has to be done at the assembly level. Pls confer with Vikram for any questions around that.